src/main/kotlin/com/openlattice/hazelcast/serializers/PrincipalTreeInvalidatedEventStreamSerializer.kt

```diff
@@ -0,0 +1,32 @@
+package com.openlattice.hazelcast.serializers
+
+import com.hazelcast.nio.ObjectDataInput
+import com.hazelcast.nio.ObjectDataOutput
+import com.openlattice.hazelcast.StreamSerializerTypeIds
+import com.openlattice.principals.PrincipalTreeInvalidatedEvent
+import org.apache.commons.lang3.RandomStringUtils
+import org.springframework.stereotype.Component
+
+@Component
+class PrincipalTreeInvalidatedEventStreamSerializer : TestableSelfRegisteringStreamSerializer<PrincipalTreeInvalidatedEvent> {
+    override fun generateTestValue(): PrincipalTreeInvalidatedEvent {
```

Comment:
Where is the implementation of `PrincipalTreeInvalidatedEvent`?

Reply:
This was removed.